app/models/manageiq/providers/ansible_tower/automation_manager/refresh_worker.rb

```diff
@@ -0,0 +1,11 @@
+class ManageIQ::Providers::AnsibleTower::AutomationManager::RefreshWorker < MiqEmsRefreshWorker
+  require_nested :Runner
+
+  def self.ems_class
+    parent
+  end
+
+  def self.settings_name
+    :ems_refresh_worker_ansible_tower_automation
```

Comment:
This implies a Settings migration is needed, as well as a change to settings.yml